src/responses/shows.js

```diff
@@ -12,6 +12,8 @@ export const NO_SHOW_FOUND = tmpl`No show found for ${0}`;
 /** ${0: Show Name} */
 export const NO_SHOW_QUEUED = tmpl`Couldn't find ${0} queued for download.`;
 
+export const NO_SHOW_SLOT = tmpl`Sorry, I couldn't understand the movie or show title I heard. Please try again with different phrasing.  Please specify show slash series, or movie slash film before the title`;
```

Comment:
I use a max line length of 100 characters, enforced with linting. This is gonna fail ci.